resample/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+resample package.
+"""
+
+__version__ = "0.21"
```

Comment:
```suggestion
__version__ = "1.0.0"
```

Just update this now?

Reply:
I usually increase the version number when I make a new release. If I am happy with the code state of the repository for a release, I make a special release PR in which I only increase the version number and write the changelog. For me this has dedicated advantages, which do not matter here. Can we perhaps jump to 0.9 instead, until we are completely finished with the interface update? There are two more things to settle
- the naming of the functions in `empirical.py`
- how to support precomputed pre-computed bootstrap and jackknife estimates

Reply:
Actually, that this version string in `__init__.py` is still there was a mistake, I fixed that now. I also get why we cannot switch to 0.9, the last release you made on PyPI was 0.21, which seems to be a typo, but it will rank higher than 0.9, I presume...

Reply:
Implemented.